src/containers/quiz/question-comment/QuestionComment.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react'
```

Comment:
```suggestion
import { useState } from 'react'
```
Importing `React` is not necessary, you can remove it.

Reply:
Done.